src/frontend/src/flows/register/passkey.ts

```diff
@@ -97,11 +97,32 @@ const savePasskeyTemplate = ({
 export const savePasskeyPage = renderPage(savePasskeyTemplate);
 
 // Prompt the user to create a WebAuthn identity or a PIN identity (if allowed)
-export const savePasskeyOrPin = async (): Promise<
-  IIWebAuthnIdentity | undefined
-> => {
+export const savePasskeyOrPin = async ({
```

Comment:
This was reverted to the old logic if `pinAllowed` true and skip to render the page if not enabled.